metagraph/src/graph/representation/succinct/dbg_succinct.hpp

```diff
@@ -110,7 +110,12 @@ class DBGSuccinct : public DeBruijnGraph {
     virtual uint64_t num_nodes() const override final;
 
     virtual void mask_dummy_kmers(size_t num_threads, bool with_pruning) final;
+
+    // If a mask is defined, return a const pointer to it. Otherwise, return nullptr
```

Comment:
remove all words that are not necessary to keep it as short as possible
```suggestion
    // Return a pointer to the mask, or NULL if not initialized
```